### PR TITLE
Empty states consistency

### PR DESCRIPTION
**Pull request by SamTremko**

### CODE REVIEW

**Comment by SamTremko** [comment]
* **#1049** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/1049?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 2  dependent PRs ([#1050](https://github.com/vexl-it/vexl/pull/1050) <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/1050?utm_source=stack-comment-icon" target="_blank">Graphite</a>, [#1051](https://github.com/vexl-it/vexl/pull/1051) <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/1051?utm_source=stack-comment-icon" target="_blank">Graphite</a>) 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @SamTremko and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/mobile/src/components/SetContactsScreen/components/ContactListSelect/index.tsx [hunk 1]
```diff
@@ -44,6 +45,7 @@ function ContactsListSelect({
   useEffect(() => {
     return () => {
       resolveAllContactsAsSeen()
+      void areContactsPermissionsGranted()
```
Comment (kaladivo) [inline apps/mobile/src/components/SetContactsScreen/components/ContactListSelect/index.tsx:48]: Nothing done with the result. Should the screen react to contacts not being grated here?

Comment (kaladivo) [inline apps/mobile/src/components/SetContactsScreen/components/ContactListSelect/index.tsx:48]: Remove

**Review by kaladivo — APPROVED** [review]

**Comment by SamTremko** [comment]
### Merge activity

* **May 9, 10:21 AM EDT**: @SamTremko merged this pull request with [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/1049).

**Pull request merged**